lms/djangoapps/grades/tests/test_models.py

```diff
@@ -271,21 +273,71 @@ def setUp(self):
             ),
             "percent_grade": 77.7,
             "letter_grade": "Great job",
+            "passed": True
         }
 
     def test_update(self):
-        created_grade = PersistentCourseGrade.objects.create(**self.params)
+        created_grade = PersistentCourseGrade.update_or_create_course_grade(**self.params)
         self.params["percent_grade"] = 88.8
         self.params["letter_grade"] = "Better job"
         updated_grade = PersistentCourseGrade.update_or_create_course_grade(**self.params)
         self.assertEqual(updated_grade.percent_grade, 88.8)
         self.assertEqual(updated_grade.letter_grade, "Better job")
         self.assertEqual(created_grade.id, updated_grade.id)
 
+    def test_passed_timestamp(self):
+        # When no letter grade exists, passed_timestamp is None
```

Comment:
Nit: Update the docstring so it refers to the `passed` boolean instead of the `letter_grade` value.